usaspending_api/etl/helpers.py

```diff
@@ -1,9 +1,18 @@
 from datetime import datetime
+import warnings
 
 from django.db.models import Q
+from django.core.cache import caches, CacheKeyWarning
 
 from usaspending_api.references.models import Agency, Location, RefCountryCode
 
+warnings.simplefilter("ignore", CacheKeyWarning)
+
+
+def clear_caches():
+    for cache in ['locations', 'recipients', 'summary-awards']:
```

Comment:
A very minor nit (and I know this is coming from the loader code itself)...the name `summary-award` is still hanging around from the days before we renamed the `SummaryAward` model to just `Award`. Do you think it's worth starting to make our code match that new name where we can?